source/tutorials/module-system/module-system.md

```diff
@@ -606,34 +596,49 @@ index 1c9a043..53860f1 100644
        };
      };
 -  };
+```
+
+In this case, you don't easily have access to the name from the marker submodules `label` option, where you otherwise could set a `default` value.
+
+Instead you can use the `config` section of the `user` submodule to set a default, like so:
+
+```diff
+# marker.nix
 +
 +    config = {
 +      departure.style.label = lib.mkDefault
 +        (firstUpperAlnum name);
 +    };
 +  });
- 
+
  in {
- 
+
 ```
 
-## Marker colors
+:::{note}
+Module options have a *precedence*, represented as an integer, which determines the priority of setting the option to a particular value.
```

Comment:
Lowest precedence wins, right? Could be worth mentioning.

Reply:
```suggestion
Module options have a *precedence*, represented as an integer, which determines the priority of setting the option to a particular value.
When merging values, the lowest precedence wins.
```